__std__/plutus-apps/library/combine-haddock.nix

```diff
@@ -39,13 +38,15 @@ cell.library.pkgs.runCommand "combine-haddock"
   mkdir -p "$out/share/doc"
 
   for pkg in $hsdocsRec; do
-    files=($pkg/share/doc/*)
-    if [ ''${#files[@]} -gt 0 ]; then
-      cp -R ''${files[@]} "$out/share/doc"
+    if [ -d "$pkg/share/doc/ghc" ]; then
```

Comment:
It's a total mystery why the `cd` command fails when copying the `ghc` docs specifically.
```
cp: cannot create regular file '/nix/store/24clzi703kn3jckwvprb45j2pdhviva1-combine-haddock/share/doc/ghc/html/Debugger.html': Permission denied
cp: cannot create regular file '/nix/store/24clzi703kn3jckwvprb45j2pdhviva1-combine-haddock/share/doc/ghc/html/DriverBkp.html': Permission denied
cp: cannot create regular file '/nix/store/24clzi703kn3jckwvprb45j2pdhviva1-combine-haddock/share/doc/ghc/html/synopsis.png': Permission denied
cp: cannot create regular file '/nix/store/24clzi703kn3jckwvprb45j2pdhviva1-combine-haddock/share/doc/ghc/html/linuwial.css': Permission denied
cp: cannot create regular file '/nix/store/24clzi703kn3jckwvprb45j2pdhviva1-combine-haddock/share/doc/ghc/html/haddock-bundle.min.js': Permission denied
cp: cannot create regular file '/nix/store/24clzi703kn3jckwvprb45j2pdhviva1-combine-haddock/share/doc/ghc/html/quick-jump.css': Permission denied
cp: cannot create regular file '/nix/store/24clzi703kn3jckwvprb45j2pdhviva1-combine-haddock/share/doc/ghc/html/quick-jump.min.js': Permission denied
cp: cannot create regular file '/nix/store/24clzi703kn3jckwvprb45j2pdhviva1-combine-haddock/share/doc/ghc/html/meta.json': Permission denied
cp: cannot create regular file '/nix/store/24clzi703kn3jckwvprb45j2pdhviva1-combine-haddock/share/doc/ghc/html/ghc.txt': Permission denied
cp: cannot create directory '/nix/store/24clzi703kn3jckwvprb45j2pdhviva1-combine-haddock/share/doc/ghc/html/src': Permission denied
```